build.cake

```diff
@@ -90,15 +90,11 @@ Task("BuildArtifacts")
                     fileNames.Add(fileName);
                 }
 
-                // DnsClient.dll is needed by Sandcastle
+                // add additional files needed by Sandcastle
                 if (targetFramework == "net452" && project == "MongoDB.Driver.Core")
                 {
                     fileNames.Add("DnsClient.dll");
-                }
-
-                // SharpCompress.dll is needed by Sandcastle
-                if (targetFramework == "net452" && project == "MongoDB.Driver.Core")
-                {
```

Comment:
The two if statements can be combined, and one additional file needs to be copied.